examples/sph/run_dustydiffuse_tva.py

```
@@ -278,13 +278,13 @@ def analytic_dsdt(t):
 
 # If the animation is not returned only a static image will be shown in the doc
 
-glob_str = os.path.join("_to_trash", "dump_dustydiffuse_tvi_*.png")
+glob_str = os.path.join("_to_trash", "dump_dustydiffuse_tva_*.png")
 ani = show_image_sequence(glob_str)
 
 from matplotlib.animation import PillowWriter
 
 writer = PillowWriter(fps=15, metadata=dict(artist="Me"), bitrate=1800)
```

Ruff (0.15.21)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)